src/gfn/utils/training.py

```diff
@@ -91,6 +91,7 @@ def states_actions_tns_to_traj(
     states_tns: torch.Tensor,
     actions_tns: torch.Tensor,
     env: DiscreteEnv,
+    conditioning: torch.Tensor | None = None,
```

Comment:
@saleml was the fix simply to account for conditioning? Otherwise I don't see what you did, other than changing the function call to a method call.